spec.html

```diff
@@ -13428,6 +13479,19 @@ <h1>Static Semantics: Contains</h1>
           1. If _symbol_ is an |Identifier| and StringValue of _symbol_ is the same value as the StringValue of |IdentifierName|, return *true*.
           1. Return *false*.
         </emu-alg>
+        <emu-grammar>OptionalChain : `?.` IdentifierName</emu-grammar>
+        <emu-alg>
+          1. If _symbol_ is a |ReservedWord|, return *false*.
+          1. If _symbol_ is an |Identifier| and StringValue of _symbol_ is the same value as the StringValue of |IdentifierName|, return *true*.
```

Comment:
If https://github.com/tc39/ecma262/pull/1519 lands before this PR, this step will need to be removed both places it occurs.